### DISCUSSION TITLE
Giving the user info about where they currently are

### DISCUSSION THREAD

**Discussion by BeksOmega** [RaspberryPiFoundation/blockly-keyboard-experimentation#6] [General]
How might we help the user orient themselves if they get lost in a workspace? There's a theory that it would be helpful to have some kind of "announcing" keybind that users can press to give them information.

1) Should this act as a replacement for navigating to figure out where you are? Or should it complement that?
2) What should it say, and in what order? E.g. "on the second else of an if block, two levels deep in a forever loop" or "inside a forever loop inside of a while loop on the second else of an if block."

**Comment by rachel-fenichel** [comment]
I think we should also have a way for the user to find their cursor if they've lost track of it. For instance, in windows you can [turn on a setting to circle your mouse pointer when you hit Ctrl](https://support.microsoft.com/en-us/windows/find-your-mouse-pointer-fast-dbc1d222-778c-da15-5218-cb8336074554).

Concretely, I would like to use this when the user tabs to the Blockly workspace for the first time: the cursor should appear in a reasonable location and should do something to draw attention. This should also happen when the user shifts focus back to the Blockly workspace after being away from it for some reason.
